plugins/dominant-color-images/tests/test-dominant-color.php

```diff
@@ -189,11 +189,11 @@ public function data_dominant_color_img_tag_add_dominant_color_requires_proper_q
 			),
 			'single quotes'         => array(
 				'image'    => "<img src='%s'>",
-				'expected' => false,
+				'expected' => true,
 			),
 			'escaped double quotes' => array(
 				'image'    => '<img src=\"%s\">',
-				'expected' => false,
+				'expected' => true,
 			),
```

Comment:
This is a strange test case to begin with. If the string is `<img src=\"http://example.com/foo.jpg\">`, the HTML Tag Processor would parse the `src` as having the value of `\"http://example.com/foo.jpg\"`. I guess this doesn't even matter because we're not doing anything with the `src` anyway. I just wonder what value this test case has in the first place.

Reply:
I think we should delete this test case, I agree it doesn't make sense.